### PR TITLE
NXP-28726: bump woodstox library version (10.10)

### PR DESCRIPTION
**Pull request by NourNuxeo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3053,6 +3053,17 @@
...
+        <groupId>org.codehaus.woodstox</groupId>
+        <artifactId>stax2-api</artifactId>
+        <version>4.2</version>
+        <exclusions>
+          <exclusion>
+            <artifactId>stax-api</artifactId>
+            <groupId>javax.xml.stream</groupId>
+          </exclusion>
+        </exclusions>
+      </dependency>
```
Comment (efge) [inline pom.xml:3066]: Could you move this elsewhere that the Jackson section please, probably next to the `stax-api` dependencies.

File: pom.xml [hunk 2]
```diff
@@ -3053,6 +3053,17 @@
...
+      <dependency>
+        <groupId>org.codehaus.woodstox</groupId>
+        <artifactId>stax2-api</artifactId>
+        <version>4.2</version>
+        <exclusions>
+          <exclusion>
+            <artifactId>stax-api</artifactId>
+            <groupId>javax.xml.stream</groupId>
+          </exclusion>
+        </exclusions>
```
Comment (efge) [inline pom.xml:3065]: I don't think that this exclusion is needed, `stax2-api` doesn't depend on `stax-api`

File: pom.xml (hunk 1 above)
Comment (NourNuxeo) [reply pom.xml]: Done ! Now near stax-api

File: pom.xml (hunk 2 above)
Comment (NourNuxeo) [reply pom.xml]: Removed also an exclusion of stax in woodstox core I was adding but not needed

**Comment by NourNuxeo** [comment]
TP OK 🟢 
https://qa2.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-nalkotob-10.10/147/console

**Pull request merged**